### ISSUE TITLE
Improve bulk uploader error messages

### ISSUE THREAD

**Issue by lg-king** [CDCgov/prime-simplereport#4556]
## Description 
We want to help users correct errors by showing them what the correct value should be and linking to the appropriate documentation. 

## Acceptance criteria
- [ ] Error message includes link to relevant field in the guide
- [ ] Error message shows example of correct format 
- [ ] Table headers match SR style and accurately describe content 

## Design considerations
- Do we want to link to the section in the guide? For all or some fields?
- What do we want to include in the error message?
- How should error messages be formatted?

Status update 2/13/23: Targeting completion 2/14 or 2/15

**Comment by vz3** [comment]
Current state:
![image.png](https://images.zenhubusercontent.com/57322f73bf93871ccd9ad030/e74675cf-b4c6-4133-824a-d32ecb88b32e)

**Comment by vz3** [comment]
Going to make some proposals for copy changes for review by @lauraykerr 
Error description > Error
patient_county is a  required column > `patient_county` is a required column (monospace field name)
Location of error > Location
Row(s): > Row:

Additionally, I'll mock up how we might lay out additional guidance.

**Comment by vz3** [comment]
Rough sketch for feedback:
![image.png](https://images.zenhubusercontent.com/57322f73bf93871ccd9ad030/751df757-5e1b-4be8-baf2-5896afa2cbd2)

Some questions to consider:
- Should all of the messaging go in the same column? Or should we have an additional column for information other than the error?
- How should the column headers be labeled?
- Is the tabular error presentation working?

**Comment by lauraykerr** [comment]
@vz3 Thanks for starting this off! Have a suggested next iteration below that responds to your questions and a couple other bits of feedback, specifically: 
- Pulled out guidance into a third column since it reads a little more cleanly.
- Labeled columns "Error," "Row(s)," and "Guidance." 
   - For "row(s)," would we ever display a location other than row number (e.g. "header" — or would this just be row 1)?
   - Not set on "guidance," and also thinking about "recommendation," "resolution," or something else 
- Rolling with tabular format for now, but might be helpful to user test
- Pared down overarching error message ("file not accepted") since the last sentence repeats the header

Would love to pair on next steps when you're back. 

<img width="622" alt="Screen Shot 2023-01-05 at 11 38 36 AM" src="https://user-images.githubusercontent.com/22102190/210835474-66cea3b0-0f1a-400c-9acf-d37d44f0bff4.png">

**Comment by lauraykerr** [comment]
A couple other thoughts and considerations: 

- Can we link directly to specific fields in the data guide from the guidance column? 
- We should include actionable guidance within the error messages to the extent possible, but where does it make sense to just link out to complete field information in the data guide?

**Comment by vz3** [comment]
> - Can we link directly to specific fields in the data guide from the guidance column?
Yes, but we'll need to verify with devs that each section is set up so we can anchor to them directly.
> We should include actionable guidance within the error messages to the extent possible, but where does it make sense to just link out to complete field information in the data guide?
Yes that's what I was struggling with in my example. I would think we link out when there's more than a sentence or two worth of guidance.

**Comment by vz3** [comment]
Your sketch looks great btw, no notes from me for now

**Comment by vz3** [comment]
Next task is for me to mock this up in higher fidelity in Figma

**Comment by vz3** [comment]
Updated comp after paring session w @lauraykerr 
https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=14722%3A42587&t=Uf1ZWXi4gHsWb6XD-1

**Comment by vz3** [comment]
This is ready for dev

**Comment by lg-king** [comment]
@vz3 update width to accommodate row numbers

**Comment by vz3** [comment]
Additionally, I want to update the width on this to be 12 columns

**Comment by vz3** [comment]
#5067 is dependent upon this story

**Comment by vz3** [comment]
Took another look at this and I think the latest comp is in a good place. cc @lauraykerr @kenieh 
https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site-(2020-2022)?node-id=15093%3A43834&t=DlznHmfkvqDFC7LQ-1
